crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -2748,6 +2778,107 @@ MAP
   -------------------
 ENDMAP
 
+NAME:   pdpol_lair_dawg_pound
+TAGS:   no_monster_gen
+DEPTH:  Lair
+KMONS:  1 = wolf / warg w:5
+KMONS:  2 = hell hound / raiju
```

Comment:
This vault steps on the dogs lair ending a bit imo. Maybe downweighting the raiju is enough but it feels like it should get something extra to make it feel more distinctive.

Reply:
Ahh yeah I thought that might be an issue... I'm definitely open to ideas for something extra, and honestly I'm not very attached to the dog theme either, I'm mostly trying to figure out fun ways to get some encounters with non-lair beast type monsters that aren't lair ends.

So a different kind of direction - do you think it would be too out there to have a hog themed minivault with a hell hog boss or two? That's the kind of thing I'm thinking, but I feel a bit out of my element when it comes to balance and reaching outside of the branch-appropriate monster set.

Reply:
Single hell hog is maybe ok? They're speed 14 and have a 3d20 fireball, so it stretches the limit of what you can put in lair (compare fire dragon which has 3d24 breath and much better defenses, but is speed 10). This would let you add the fabled holy swine though...